components/util/opts.rs

```diff
@@ -188,6 +188,9 @@ pub struct Opts {
 
     /// Do not use native titlebar
     pub no_native_titlebar: bool,
+
+    //This variable is of the type enum Render API defined below, used for selecting graphic options dynamically
```

Comment:
This should be a doc comment (3 slashes) - something like "Select GLX or EGL backend (Linux only)"